src/Calculator/RefactoredTokenizer.js

```diff
@@ -0,0 +1,67 @@
+function Tokenizer() {
+  function ValueToken(value, type) {
+    this.type = type;
+    this.value = Number(value);
+  }
+
+  function OperationToken(value1, value2, type) {
+    this.type = type;
+    this.value1 = value1;
+    this.value2 = value2;
+  }
+
+  const tokenize = input => {
+    const array = input.match(/(\d+)?(\.?\d+)|\+|\-|\*|\//g);
+    const symbol = {
```

Comment:
This sympols map could be defined outside of the function. This way it does not gets re-defined every time you call the function.